packages/webapp/src/pages/Connection/components/SyncRow.tsx

```diff
@@ -134,6 +134,7 @@ export const SyncRow: React.FC<{ sync: SyncResponse; connection: Connection; pro
                 </Link>
             </Table.Cell>
             <Table.Cell bordered>{formatFrequency(sync.frequency)}</Table.Cell>
+            <Table.Cell bordered>{sync.object_count ?? '-'}</Table.Cell>
```

Comment:
should we format the count somewhat? How does it look in the UI when count is big?